src/features/events/components/EventOverviewCard/index.tsx

```diff
@@ -237,14 +638,14 @@ const EventOverviewCard: FC<EventOverviewCardProps> = ({
                   multiline
                   onChange={(ev) => setInfoText(ev.target.value)}
                   rows={4}
-                  sx={{ marginBottom: 2 }}
+                  sx={{ marginBottom: 2, marginLeft: 1, paddingRight: 2 }}
                   value={infoText}
                 />
               )}
               renderPreview={() => {
                 if (eventData.info_text !== '') {
                   return (
-                    <Box>
+                    <Box mb={2} ml={2}>
```

Comment:
Why are there only margins on the left and bottom sides? I would say that in general margins should be consistent around a block. There are of course many exceptions, but when design (styling) and semantics (HTML/DOM) align well, it should be rare to have to have different margins on different sides.

I would just do `<Box m={2}>` in this case, unless you know of some reason not to?